tests/hydra/test_hydra_runner.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import subprocess
+import sys
+from os import path
+
+import pytest
+
+
+class TestHydraRunner:
```

Comment:
Can you add a check that checks if no `.hydra` folders are created?

Reply:
good idea, will do

Reply:
Refined two tests, so they check the absence of both `.hydra` dir default log file.
When reverted one or the other change, the tests fail:
```
tests/hydra/test_hydra_runner.py::TestHydraRunner::test_no_config PASSED                                                                                                                             [ 16%]
tests/hydra/test_hydra_runner.py::TestHydraRunner::test_config1 FAILED                                                                                                                               [ 33%]
tests/hydra/test_hydra_runner.py::TestHydraRunner::test_config1_invalid PASSED                                                                                                                       [ 50%]
tests/hydra/test_hydra_runner.py::TestHydraRunner::test_config2 FAILED                                                                                                                               [ 66%]
tests/hydra/test_hydra_runner.py::TestHydraRunner::test_config2_invalid PASSED                                                                                                                       [ 83%]
tests/hydra/test_hydra_runner.py::TestHydraRunner::test_config2_filepath_schema PASSED                                                                                                               [100%]
```